utils/dates.py

```diff
@@ -23,6 +23,59 @@
     ["RRULE:FREQ=WEEKLY;UNTIL={}".format(MID_TERM_BEGIN.strftime("%Y%m%dT080000Z"))],
 ]
 
+
+### getting holidays
+def get_holidates() -> (list[datetime], list[str, datetime]):
+    """
+    scrapes holiday list from IITKGP website
+    returns: list of holidays as occasions and datetime objects
+    """
+    url = "https://www.iitkgp.ac.in/holidays?lang=en"
```

Comment:
This link needs to be changed - `?lang=en` query has to be removed. 
Since Ojas is not available today, I'm merging the PR and fixing it myself